### ISSUE TITLE
Vim mode and full screen escaping all the time

### ISSUE THREAD

**Issue by Spectralgo** [Razumihin/obsidian-fullscreen-plugin#2]
Hi thanks for your work, I enjoy it very much. 

I'm using obsidian's vim mode and I would like to change "esc" to another key binding. I don't know if you know vim, but you are using the escape key all the time to edit.

Furthermore, I can't really edit in full screen, if I'm exiting the full screen mode each time I type on escape.

Could you help me?

Thanks.

**Comment by Razumihin** [comment]
Yeah, I know it's not nice when writing in Vim mode (I use it myself too), but actually the esc seems to be hardwired in the layout system; Either the obsidian one or Electron itself. I made this plugin with pretty minimal effort for reading the notes and therefore it just uses the underlying layout system for the action. I'll try to look into this later.

**Comment by droctothorpe** [comment]
+1 to implementing this.

**Comment by fergusj** [comment]
+1 to implementing this.

**Comment by Archie-2021** [comment]
Hope you change this issue sooner as it is a awesome plugin like vim.
